lib/zendesk_api/middleware/request/etag_cache.rb

```diff
@@ -1,4 +1,5 @@
 require "faraday/middleware"
+require "active_support/notifications"
```

Comment:
We should be able to remove this since we're passing in `options[:instrumentation]` and don't use the module directly.